Apps/Common/SIMCoupledSI.h

```diff
@@ -62,6 +85,29 @@ class SIMCoupledSI : public SIMCoupled<T1,T2>
 
       if ((conv = this->checkConvergence(tp,status1,status2)) <= SIM::DIVERGED)
         return false;
+
+      // Calculate aitken acceleration factor
+      if (aitken && omega0 != 0.0) {
+        if (tp.iter > 0) {
+            Vector r1 = this->getAitkenResidual();
+            r1 -= prevRes;
+            omega *= -prevRes.dot(r1) / r1.dot(r1);
+        }
+        prevRes = this->getAitkenResidual();
+      }
+
+      // Perform relaxation
+      if (omega0 != 0.0) {
```

Comment:
This if is not necessary - the block can be included in the previous `if (omega0 != 0.0)`

Reply:
no, because you can do relaxation without aitken.